.ci_support/build_all.py

```diff
@@ -44,17 +45,32 @@ def build_all(recipes_dir, arch):
     script_dir = os.path.dirname(os.path.realpath(__file__))
     variant_config_file = os.path.join(script_dir, "{}.yaml".format(get_config_name(arch)))
 
+    has_meta_yaml = False
+    has_recipe_yaml = False
+
     found_cuda = False
     found_centos7 = False
     for folder in folders:
         meta_yaml = os.path.join(recipes_dir, folder, "meta.yaml")
         if os.path.exists(meta_yaml):
+            has_meta_yaml = True
             with(open(meta_yaml, "r", encoding="utf-8")) as f:
                 text = ''.join(f.readlines())
                 if 'cuda' in text:
                     found_cuda = True
                 if 'sysroot_linux-64' in text:
                     found_centos7 = True
+
+        recipe_yaml = os.path.join(recipes_dir, folder, "recipe.yaml")
+        if os.path.exists(recipe_yaml):
+            has_recipe_yaml = True
+            with(open(recipe_yaml, "r", encoding="utf-8")) as f:
+                text = ''.join(f.readlines())
+                if 'cuda' in text:
+                    found_cuda = True
+                if 'sysroot_linux-64' in text:
+                    found_centos7 = True
```

Comment:
Now that this format is YAML-parsable, we could improve this hack and actually analyze the dependencies :D But yea, no biggies for now.

Reply:
Yep! Will modify when CI works properly. :)

Reply:
This is a bit more involved due to outputs so Ill defer this for now.